shared/libraries/config_protocol/include/config_protocol/config_protocol.h

```diff
@@ -175,4 +175,15 @@ class RpcContext
     UserPtr user;
 };
 
+inline std::set<uint16_t> GetSupportedConfigProtocolVersions()
+{
+    return {0, 1, 2, 3, 4, 5};
+}
+
+inline constexpr uint16_t GetLatestSupportedConfigProtocolVersion()
+{
+    return 5; // *GetSupportedConfigProtocolVersions().rbegin();
+}
```

Comment:
Isn't this just "getProtocolVersion"? This will always be the greatest value in the above list.